src/vega_viewer/vega/specs/histogram.cljs

```diff
@@ -60,12 +65,24 @@
                        {:text
                         {:template
                          "{{ datum.data | number:'.3s'}}"}}})
-            spec))]
+            spec))
+        duration-chart (fn [spec]
```

Comment:
Looking at how this is being [used by the client](https://github.com/onaio/zebra/pull/5012/files#diff-f463d0d5758bf647941559e7eb3cbdcbR92), making them exclusive with a `:x-axist-tick-labels` option seems to make sense